apache-rat-core/src/main/java/org/apache/rat/document/impl/guesser/NoteGuesser.java

```diff
@@ -45,26 +52,37 @@ public class NoteGuesser {
         "THIRD_PARTY_NOTICES", "THIRD_PARTY_NOTICES.TXT",
         "COPYRIGHT", "COPYRIGHT.TXT",
         "BUILDING", "BUILDING.TXT",
-        "BUILD", "BUILT.TXT",//
+        "BUILD", "BUILT.TXT",
         "DEPENDENCIES"
     };
 
+    /**
+     * List of note file extensions.  Extensions that indicate a file is a note file.
+     */
     private static final String[] NOTE_FILE_EXTENSIONS = {
```

Comment:
Just by following the name I'd guess that extensions should mean:
* empty/none
* .txt
* .md
and we could iterate over file names and its extensions - WDYT?

Reply:
@Claudenw maybe we leave this for a different kind of refactoring, but I still find the name a bit misleading :) (but will merge nevertheless) Thx.